File: app/src/main/java/dime/android/todo/logic/Task.java
package dime.android.todo.logic;

public class Task
{
	public static final int	PRIORITY_LOW		= 0;
	public static final int	PRIORITY_NORMAL	= 1;
	public static final int	PRIORITY_HIGH		= 2;

	private int					id;
	private String				name;
	private int					priority;
	private boolean			completed;


	public Task (int id, String name, int priority, boolean completed)
	{
		this.id = id;
		this.name = name;
		this.priority = priority;
		this.completed = completed;
	}


	public int getId ( )
	{
		return id;
	}


	public void setId (int id)
	{
		this.id = id;
	}


	public String getName ( )
	{
		return name;
	}


	public void setName (String name)
	{
		this.name = name;
	}


	public int getPriority ( )
	{
		return priority;
	}


	public void setPriority (int priority)
	{
		this.priority = priority;
	}


	public boolean isCompleted ( )
	{
		return completed;
	}


	public void setCompleted (boolean completed)
	{
		this.completed = completed;
	}

}

File: app/src/main/kotlin/dime/android/todo/db/Task.kt
package dime.android.todo.db

/**
 * Represents a single Task
 *
 * Created by dime on 06/02/16.
 */
class Task(var id: Int, var name: String, var priority: Int, var completed: Int) {

    /**
     * Is the task completed?
     */
    fun isCompleted() = completed == 1

    /**
     * Equals
     */
    override fun equals(other: Any?) = if (other is Task) other.id == id else false
}
